source/developer/core-developer-handbook.md

```diff
@@ -17,8 +17,9 @@ Below is the list of core developers working on Mattermost:
 - Chris Duarte - @uberchris on [pre-release.mattermost.com](https://pre-release.mattermost.com/) and [@csduarte](https://github.com/csduarte) on GitHub
 
 Below is the list of core developers working on individual Mattermost repositories:
-- Pan Luo - [mattermost-docker](https://github.com/mattermost/mattermost-docker) - @compass on [pre-release.mattermost.com](https://pre-release.mattermost.com/) and [@xcompass](https://github.com/xcompass) on GitHub
 - Yuya Ochiai - [desktop](https://github.com/mattermost/desktop) - @yuya-oc on [pre-release.mattermost.com](https://pre-release.mattermost.com/) and [@yuya-oc](https://github.com/yuya-oc) on GitHub
+- Pan Luo - [mattermost-docker](https://github.com/mattermost/mattermost-docker) - @compass on [pre-release.mattermost.com](https://pre-release.mattermost.com/) and [@xcompass](https://github.com/xcompass) on GitHub
+- Pichou Kyâne - [mattermost-docker](https://github.com/mattermost/mattermost-docker) - @pichouk on [pre-release.mattermost.com](https://pre-release.mattermost.com/) and [@pichouk](https://github.com/xcompass) on GitHub
```

Comment:
Is this the correct GitHub link for Pichou? `[@pichouk](https://github.com/xcompass)`